src/sparseml/exporters/transforms/gemm_to_qlinearmatmul.py

```diff
@@ -93,7 +90,10 @@ def transform(self, model: ModelProto) -> ModelProto:
             output_dequant = match.children[0][1]
             if output_dequant is not None:
                 output_dequant_child = graph.get_node_single_child(output_dequant)
-                if output_dequant_child and output_dequant_child.op_type == "Gemm":
+                if output_dequant_child and output_dequant_child.op_type in {
+                    "Gemm",
+                    "Softmax",
```

Comment:
Will this adversely affect transformers? Not sure if MHA in transformers gets exported as Gemm or MatMul

Reply:
generally MatMul, also we have not been targeting output quantization there anyways